src/command_modules/azure-cli-webapp/azure/cli/command_modules/webapp/_help.py

```diff
@@ -9,5 +9,5 @@
 
 helps['appservice'] = """
     type: group
-    short-summary: Create and deploy web apps
+    short-summary: commands for managing your azure web apps, app service plans, etc
```

Comment:
Also, can we expand more on 'etc'?
Just looking at this summary, it isn't clear what's in 'etc'.